hub/core/storage/lru_cache.py

```diff
@@ -83,7 +90,15 @@ def get_cachable(
             An instance of `expected_class` populated with the data.
         """
 
-        item = self[path]
+        if url:
+            from hub.util.remove_cache import get_base_storage
```

Comment:
should we have an assert statement here if the expected class isn't BaseChunk? not sure what the behaviour will be in that case.